.github/workflows/pr-require-backport-label.yaml

```diff
@@ -0,0 +1,22 @@
+name: PR require backport label
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+    branches:
+      - master
+      - next
+jobs:
+  label:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: mheap/github-action-required-labels@v5
+        with:
+          mode: minimum
+          count: 1
+          labels: "backport/none\nbackport/.*-v?[\\d.]*$"
```

Comment:
It won't catch the `backport/2024.1` label

Reply:
I think now it's a better version of the regex, that does cover all of the options